addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6964,6 +6964,21 @@ registry.BackgroundShape = SnippetOptionWidget.extend({
                 .prepend($(`<we-colorpicker data-color="true" data-color-name="${colorName}"></we-colorpicker>`)[0]);
```

Comment:
A last comment, depending on your answer I'll r+: 
There is nothing to say about the background shapes, it seems to work perfectly :wink: 
But I noticed something wrong with the background color palettes (even on runbot so not related to your fix):
- In the Theme tab, change one of the colors
- (If you stay in the Theme tab to select the color, then there is nothing wrong)
- But if you click on the Blocks tab or on the snippet, then any color picker in the options is outdated (in the picture, the blue should be yellow)
![image](https://github.com/odoo/odoo/assets/102796037/c120e6ec-9420-4816-87ae-f8ce7a99de23)
![image](https://github.com/odoo/odoo/assets/102796037/dc6cd098-b704-46fa-9feb-c60e7a8b1610)

So I am just wondering if we should create a task for it (if so, tell me and I will do it), or if it is worth fixing it here :thinking:  I am asking because of what you did in `_onReloadBundles` -> since now there is the system to update the options in these cases, why not make the same as you did for the shapes (so the `renderCustomXML`) but for the background in general, while you are at it :slightly_smiling_face:

Reply:
I tried adding a `specialRender` to the `ColorpickerUserValueWidget` to re-render the color palette and having the base `SnippetOptionWidget`'s `updateUI` call it on every widget if it is defined, but it did not work.
It seems the CSS `var`s are not updated.
Since this seems to be a problem of a different technical nature, I think it can be addressed in its own task.